cellprofiler_core/utilities/zarr.py

```diff
@@ -204,11 +204,18 @@ def map_wells(self):
         if 'plate' not in attrs or 'wells' not in attrs['plate']:
             return False
         well_data = attrs['plate']['wells']
+        if not well_data:
+            # Empty well data for some reason
+            return False
         mapper = {}
         if 'column_index' in well_data[0]:
-            # Standard format
+            # v0.2 format
```

Comment:
For completeness, I would slightly update this coment.
https://ngff.openmicroscopy.org/0.2/index.html#plate-md does not mandate any metadata for the row/column index. The keys detected by this utility were populated by `bioformats2raw 0.4.0` 

In version 0.4 of the NGFF specification, `columnIndex/rowIndex` are  required attributes for each element of the `wells` array